src/components/dialogs/filter/name-wrapper.js

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
I used this component for the development of voltage init parameter saving and moved the file to dialogs/commons, do you agree to move it there too so we don't duplicate it ?